src/renderer/components/settings.tsx

```diff
@@ -118,12 +124,63 @@ export default function SettingsMenu({
 		if (!open) setIsRecording(false);
 	}, [open]);
 
+	useEffect(() => {
+		const fetchOpenAtLogin = async () => {
+			const isOpen = await settings.getOpenAtLogin();
+			setOpenAtLogin(isOpen);
+		};
+		fetchOpenAtLogin();
+	}, []);
+
+	useEffect(() => {
+		openAtLogin
```

Comment:
can you add this in the dependency array please?

Reply:
Added 👍